editor/editor_settings.cpp

```diff
@@ -381,7 +381,7 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 
 	// On save
 	_initial_set("filesystem/on_save/compress_binary_resources", true);
-	_initial_set("filesystem/on_save/safe_save_on_backup_then_rename", true);
+	_initial_set("filesystem/on_save/safe_save_on_backup_then_rename", false);
```

Comment:
Pretty sure I did this while trying to get editor working on Switch. Should probably be reverted.

Reply:
- [ ] false -> true